src/libcrun/utils.c

```diff
@@ -2507,3 +2507,32 @@ get_overflow_gid (void)
     }
   return gid;
 }
+
+void
+consume_trailing_slashes (char *path)
+{
+  if (! path || path[0] == '\0')
+    {
+      return;
+    }
+
+  // Handle the case where the path is just a single slash.
+  if (strcmp (path, "/") == 0)
+    {
+      return;
+    }
```

Comment:
This probably does not belong here, as it is a utility function.

Why? The handling of a single slash is a use case specific to the code that calls this function.

Reply:
ok, I don't think we need to handle this condition separately. It will covered as part of the below snippet:
```go
  cleanup_free char *normalized_path = xstrdup (fullname);
  consume_trailing_slashes (normalized_path);
  if (normalized_path[0] == '\0')
    strcpy (normalized_path, "/");
```